js/index.js

```diff
@@ -48,15 +48,36 @@ let midImage = document.getElementById("middle-img");
 midImage.setAttribute('src', siteContent["main-content"]["middle-img-src"])
 
 //Nav Items
+
+
+
+
 let nav = document.querySelectorAll('a')
 
+
 nav[0].textContent = siteContent['nav']['nav-item-1']
 nav[1].textContent = siteContent['nav']['nav-item-2']
 nav[2].textContent = siteContent['nav']['nav-item-3']
 nav[3].textContent = siteContent['nav']['nav-item-4']
 nav[4].textContent = siteContent['nav']['nav-item-5']
 nav[5].textContent = siteContent['nav']['nav-item-6']
```

Comment:
nav.forEach(e, i => 
  e.textContent = siteContent['nav'][`nav-item-${i}`]
)

might be i + 1